<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPUViewer</class>
 <widget class="QMainWindow" name="MPUViewer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>268</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MPU Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditA"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditS"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditX"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>PC</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditY"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditPC"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>81</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>P</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkboxN">
        <property name="text">
         <string>7: N</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxV">
        <property name="text">
         <string>6: V</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxB">
        <property name="text">
         <string>4: B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxD">
        <property name="text">
         <string>3: D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxI">
        <property name="text">
         <string>2: I</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxZ">
        <property name="text">
         <string>1: Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxC">
        <property name="text">
         <string>0: C</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditA</tabstop>
  <tabstop>lineEditX</tabstop>
  <tabstop>lineEditY</tabstop>
  <tabstop>lineEditS</tabstop>
  <tabstop>lineEditPC</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
